### PR TITLE
Sijuade details

### PR DESCRIPTION
**Pull request by supercede**

### CODE REVIEW

**Review by NedyUdombat — CHANGES_REQUESTED** [review]
Great job, Sijuade. Please resolve confilcts on your branch

**Comment by supercede** [comment]
> Great job, Sijuade. Please resolve confilcts on your branch

I have

File: contributors.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
 Nedy Udombat
 Desmond Edem
+
```
Comment (NedyUdombat) [inline contributors.txt]: Please remove all this extra spaces

**Review by NedyUdombat — CHANGES_REQUESTED** [review]

**Review by NedyUdombat — APPROVED** [review]
Great work

**Pull request merged**